docs/docs/expressions.md

```diff
@@ -698,16 +698,16 @@ For the hierarchy data set with the given _name_, returns the array of ancestors
 
 Functions for accessing web browser facilities.
 
-<a name="open" href="#open">#</a>
```

Comment:
The `open` function was removed in [v3.0.0-beta.32](https://github.com/vega/vega/releases/tag/v3.0.0-beta.32). This change was simply to update the (stale) documentation to match.